File: FACT_NEGOCIO/Model/SECURITYDataBaseModel.cs
using CAPA_DATOS;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
namespace DataBaseModel {
   public class Security_Permissions : EntityClass {
       [PrimaryKey]
       public int? Id_Permission { get; set; }
       public string? Descripcion { get; set; }
       public string? Estado { get; set; }
   }
   public class Security_Roles : EntityClass {
       [PrimaryKey]
       public int? Id_Role { get; set; }
       public string? Descripcion { get; set; }
       public string? Estado { get; set; }
   }
   public class Security_Users : EntityClass {
       [PrimaryKey]
       public int? Id_User { get; set; }
       public string? Nombres { get; set; }
       public string? Estado { get; set; }
       public string? Descripcion { get; set; }
       public string? Password { get; set; }
       public string? Mail { get; set; }
       public string? Token { get; set; }
       public DateTime? Token_Date { get; set; }
       public DateTime? Token_Expiration_Date { get; set; }
   }
   public class Security_Permissions_Roles : EntityClass {
       [PrimaryKey]
       public int? Id_Role { get; set; }
       [PrimaryKey]
       public int? Id_Permission { get; set; }
       public string? Estado { get; set; }
   }
   public class Security_Users_Roles : EntityClass {
       [PrimaryKey]
       public int? Id_Role { get; set; }
       [PrimaryKey]
       public int? Id_User { get; set; }
       public string? Estado { get; set; }
   }
}

File: FACT/wwwroot/FrontModel/SECURITYDataBaseModel.js
import { EntityClass } from "../WDevCore/WModules/EntityClass.js";
import { WAjaxTools } from "../WDevCore/WModules/WComponentsTools.js";
class Security_Permissions extends EntityClass {
   constructor(props) {
       super(props, 'EntitySECURITY');
   }
   Id_Permission = { type: 'number', primary: true };
   Descripcion = { type: 'text' };
   Estado = { type: 'text' };
}
export { Security_Permissions }
class Security_Permissions_Roles extends EntityClass {
   constructor(props) {
       super(props, 'EntitySECURITY');
   }
   Estado = { type: 'text' };
}
export { Security_Permissions_Roles }
class Security_Roles extends EntityClass {
   constructor(props) {
       super(props, 'EntitySECURITY');
   }
   Id_Role = { type: 'number', primary: true };
   Descripcion = { type: 'text' };
   Estado = { type: 'text' };
}
export { Security_Roles }
class Security_Users extends EntityClass {
   constructor(props) {
       super(props, 'EntitySECURITY');
   }
   Id_User = { type: 'number', primary: true };
   Nombres = { type: 'text' };
   Estado = { type: 'text' };
   Descripcion = { type: 'text' };
   Password = { type: 'text' };
   Mail = { type: 'text' };
   Token = { type: 'text' };
   Token_Date = { type: 'date' };
   Token_Expiration_Date = { type: 'date' };
}
export { Security_Users }
class Security_Users_Roles extends EntityClass {
   constructor(props) {
       super(props, 'EntitySECURITY');
   }
   Estado = { type: 'text' };
}
export { Security_Users_Roles }
